### PR TITLE
Add anaglyph sensor

### PR DESCRIPTION
**Pull request by asafMasa**

### CODE REVIEW

File: src/app/@ansyn/ansyn/components/overlay-out-of-bounds/overlay-out-of-bounds.component.spec.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { geometry } from '@turf/turf';
 import { Store, StoreModule } from '@ngrx/store';
 import { TranslateModule } from '@ngx-translate/core';
 
-describe('OverlayOutOfBoundsComponent', () => {
+describe('AnaglyphSensorAlertComponent', () => {
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/components/overlay-out-of-bounds/overlay-out-of-bounds.component.spec.ts]: revert

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/anaglyph-sensor/alert-component/anaglyph-sensor-alert.component.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	styleUrls: ['./anaglyph-sensor-alert.component.less']
+})
+@AutoSubscriptions()
+export class AnaglyphSensorAlertComponent implements OnInit, OnDestroy, IEntryComponent {
+	@Input() mapId: string;
+	mapState: IMapSettings;
+	mode: AnaglyphComponentMode;
+
+	@AutoSubscription
+	mapState$: Observable<Dictionary<IMapSettings>> = this.store.pipe(
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/anaglyph-sensor/alert-component/anaglyph-sensor-alert.component.ts]: select by id

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/anaglyph-sensor/alert-component/anaglyph-sensor-alert.component.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+		tap((maps) => this.mapState = maps[this.mapId])
+	);
+
+	ShowAnaglyph(): void {
+		this.anaglyphSensorService.displayAnaglyph(this.mapState, this.mapState.data.overlay).pipe(take(1)).subscribe((res) => {
+			if (res) {
+				this.mode = AnaglyphComponentMode.ShowOriginal;
+			}
+		});
+		// const communicator = this.communicatorService.provide(this.mapState.id);
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/anaglyph-sensor/alert-component/anaglyph-sensor-alert.component.ts]: remove remarks

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/anaglyph-sensor/alert-component/anaglyph-sensor-alert.component.ts [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+	}
+
+	ngOnInit(): void {
+	}
+
+	ngOnDestroy(): void {
+	}
+
+	constructor(public store: Store<any>,
+				protected communicatorService: ImageryCommunicatorService,
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/anaglyph-sensor/alert-component/anaglyph-sensor-alert.component.ts]: remove unnecessary import

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/anaglyph-sensor/anaglyph-sensor.module.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+);
+
+@NgModule({
+	imports: [
+		CommonModule,
+		ImageryModule.provide({
+			plugins: [
+				AnaglyphSensorPlugin
+			],
+			maps: [OpenLayersMap, OpenLayersDisabledMap],
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/anaglyph-sensor/anaglyph-sensor.module.ts]: maps: []

File: src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/anaglyph-sensor/plugin/anaglyph-sensor.plugin.ts [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+		select(selectOverlayFromMap(this.mapId)),
+		tap((overlay: IOverlay) => {
+			this.overlay = overlay;
+			if (Boolean(overlay)) {
+				this.anaglyphSensorService.isSupprotedOverlay(overlay).pipe(take(1)).subscribe((isSupproted: boolean) => {
+					this.isEnabled = isSupproted;
+					if (isSupproted) {
+						this.store$.dispatch(new AddAlertMsg({ key: AlertMsgTypes.anaglyphSensor, value: this.mapId }));
+					} else {
+						this.store$.dispatch(new RemoveAlertMsg({
```
Comment (asafMasa) [inline src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/anaglyph-sensor/plugin/anaglyph-sensor.plugin.ts]: one line

File: src/app/@ansyn/ansyn/modules/alerts/model.ts [hunk 7]
```diff
@@ -1,6 +1,9 @@
 export enum AlertMsgTypes {
 	OverlaysOutOfBounds = 'overlaysOutOfBounds',
-	overlayIsNotPartOfQuery = 'overlayIsNotPartOfQuery'
+	overlayIsNotPartOfQuery = 'overlayIsNotPartOfQuery',
+	anaglyphSensor = 'anaglyphSensor'
 }
```
Comment (tltk90) [inline src/app/@ansyn/ansyn/modules/alerts/model.ts]: should we update `alertMsg` Map inside the `overlay-status.reducer` with anaglyphSensor entry?

**Pull request merged**